src/app.md

```diff
@@ -606,25 +606,30 @@ use semver::Version;
 # }
 
 fn run() -> Result<()> {
-    let mut release = Version::parse("0.2.6")?;
-    assert_eq!(release, Version {
-        major: 0,
-        minor: 2,
-        patch: 6,
-        pre: vec!(),
-        build: vec!(),
-    });
+    let version_str = "0.2.6";
```

Comment:
i see no reason for separate `version_str`.

Reply:
Hmmm you're right; I separated it to match the other example, but there's no practical reason for it.